test/integration/scheduler_perf/config/performance-config.yaml

```diff
@@ -539,6 +551,7 @@
       measurePods: 2000
   - name: 5000Nodes_5000Pods
     labels: [performance]
+    threshold: 125
```

Comment:
I looked at this one a lot... Maybe it is fine for now. Jobs don't usually have spreading, which are the ones that require more throughput.